docs/releases/1.9.txt

```diff
@@ -416,6 +416,15 @@ Miscellaneous
   Therefore, the ``@skipIfCustomUser`` decorator is no longer needed to
   decorate tests in ``django.contrib.auth``.
 
+* In Generic Class Based Views,
```

Comment:
I agree that it's more than clear from context, I tend to add 'generic' because Russell Keith-Magee always mentions in his talks that in feedback, people very often confuse CBVs with gCBVs, so it might be helpful if docs were always clear that the two are very different.

In other words, it's more about saying "reminder: these 2 are separate things" rather than saying "here we're talking about gCBV not CBV".